boards/kb2040/src/lib.rs

```diff
@@ -0,0 +1,51 @@
+#![no_std]
+
+pub use rp2040_hal as hal;
+#[cfg(feature = "rt")]
+extern crate cortex_m_rt;
+#[cfg(feature = "rt")]
+pub use cortex_m_rt::entry;
+
+pub use hal::pac;
+
+hal::bsp_pins!(
+    Gpio0 { name: d0 },
+    Gpio1 { name: d1 },
```

Comment:
Shouldn't these be Tx/Rx to match the UART default function on these pins?

Reply:
Oh yes you're right. I will fix that when I get home today. For future reference, how does one figure out the default function for each pin? The silkscreen says Tx/Rx which I overlooked. I took a peak at section 1.4 of the datasheet, but I don't see anything about default pin functions.

Reply:
I meant default to mean the recommended use as per the board markings.
I usually pick 0 and 1 for UART TX/RX because they're easy to locate on a pico.

To work out the pin default, look at the reset value of funcsel and the "function select" page of the datasheet.
I believe their default function is to not have a special function.